istioctl/pkg/analyze/analyze.go

```diff
@@ -359,6 +365,8 @@ func Analyze(ctx cli.Context) *cobra.Command {
 	analysisCmd.PersistentFlags().StringArrayVar(&remoteContexts, "remote-contexts", []string{},
 		`Kubernetes configuration contexts for remote clusters to be used in multi-cluster analysis. Not to be confused with '--context'. `+
 			"If unspecified, contexts are read from the remote secrets in the cluster.")
+	analysisCmd.PersistentFlags().StringArrayVarP(&selectedAnalyzers, "analyzer", "a", []string{},
```

Comment:
Can you add an example to the command's example? Also, can you add a test for this new flag in https://github.com/istio/istio/blob/master/tests/integration/pilot/analyze_test.go?

Reply:
BTW we do have a capitalized flag `-A` for all namespaces, this short flag `a` may be confused for some users.

Reply:
updated

Reply:
@zirain sorry I mean the example here https://github.com/istio/istio/blob/a8c26da5598f180e22d5d77df0ca95e9016459a2/istioctl/pkg/analyze/analyze.go#L97-L117. Probably add the example under `# List available analyzers`